week-1/exercises/initials.js

```diff
@@ -4,3 +4,20 @@ let lastName = "Johnson";
 
 // Declare a variable called initials that stores the first character of each string in upper case to form the user's initials
 // Log the variable in each case
+
+const firstNameInitial = firstName.charAt(0).toUpperCase();
+
+console.log(firstNameInitial);
+
+const middleNameInitial = middleName.charAt(0).toUpperCase();
+
+console.log(middleNameInitial);
+
+const lastNameInitial = lastName.charAt(0).toUpperCase();
+
+console.log(lastNameInitial);
+
+const initials =
+  firstNameInitial + "." + middleNameInitial + "." + lastNameInitial + ".";
+
+console.log(initials);
```

Comment:
This is correct & gets the right answer. Good use of 'toUpperCase' for completeness, but is it perhaps redundant in this situation?

Reply:
Thanks for your feedback. You're absolutely right that, in this case, using the "toUpperCase()" method may be redundant since the first letter of names is typically in capital letters. I included it as a precaution in case a lowercase letter is mistakenly input. However, I appreciate the insight and will consider simplifying the code for situations where the input is expected to be in uppercase. Your guidance is valuable.